SEED-platform/seed
# Hannah/analysis

#### Any background context you want to provide?

#### What's this PR do?
Creates Element Statistic analysis

#### How should this be manually tested?
1. go to a property with scope_one_emission_codes elements
2. run a "Element Statistic" analysis.
3. See the newly added columns. They will be the name of the elements 

#### What are the relevant tickets?


#### Screenshots (if appropriate)
<img width="1792" alt="image" src="https://github.com/user-attachments/assets/098fbdae-afaa-4bf5-b9dd-c36fb1c075c2">
<img width="1792" alt="image" src="https://github.com/user-attachments/assets/2d6e3449-2a4c-4638-af02-095fed8c4797">
